tests/ssp/host/host.cpp

```diff
@@ -0,0 +1,132 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include <openenclave/host.h>
+#include <openenclave/internal/error.h>
+#include <openenclave/internal/tests.h>
+#include <cassert>
+#include <cstdio>
+#include <cstdlib>
+#include <cstring>
+#include "ssp_u.h"
+
+// uint64_t host_my_ocall(uint64_t val)
```

Comment:
Why is there commented out code?   Recommend removing all commented out code.

Reply:
Removed.